package.json

```diff
@@ -1,27 +1,28 @@
 {
   "name": "resin-semver",
   "version": "0.1.1",
-  "description": "resin specific semver utility methods",
+  "description": "Resin.io specific semver utility methods",
   "main": "src/index.js",
   "typings": "src/index.d.ts",
   "scripts": {
     "test": "mocha -r ts-node/register --outDir tmp test/index.spec.ts",
-    "build": "tsc src/index.ts"
+    "build": "tsc src/index.ts",
+    "ci": "npm run test"
   },
   "author": "Lucian Buzzo <lucian@resin.io>",
   "license": "Apache-2.0",
   "dependencies": {
-    "lodash.memoize": "^4.1.2",
-    "semver": "^5.3.0"
+    "lodash.memoize": "4.1.2",
+    "semver": "5.3.0"
   },
   "devDependencies": {
-    "@types/chai": "^4.0.1",
-    "@types/lodash.memoize": "^4.1.2",
-    "@types/mocha": "^2.2.41",
-    "@types/semver": "^5.3.32",
-    "chai": "^4.1.0",
-    "mocha": "^3.4.2",
-    "ts-node": "^3.2.1",
-    "typescript": "^2.4.2"
+    "@types/chai": "4.0.1",
+    "@types/lodash.memoize": "4.1.2",
+    "@types/mocha": "2.2.41",
+    "@types/semver": "5.3.32",
+    "chai": "4.1.0",
+    "mocha": "3.4.2",
+    "ts-node": "3.2.1",
+    "typescript": "2.4.2"
```

Comment:
I don't think we should do this, because there's some downsides, it's inconsistent with most/all our other modules, and there's a better option.

Practically, this is going to mean npm can't dedupe dependencies (i.e. if this depends on semver 5.3.0 and another module uses semver 5.3.1, both will have to be installed and webpack bundled). That's bad for bundle and install sizes, but if you have slightly different behaviour between the two modules it can lead to all sorts of crazy fun bugs where things only break in half your application.

More fluffily, while breaking patch releases are super annoying, it's generally better to enforce any version fixing we need at the top level (i.e. the UI). There we can use something that works recursively, which is more reliable and easier to manage. If we want to fix versions just for development on this module itself here, but not for downstream consumers, we should use npm5 or yarn, which can happily do that.